docker-compose.dev.yml

```diff
@@ -92,15 +92,14 @@ services:
       - ENCRYPTION_PRIMARY_KEY=${ENCRYPTION_PRIMARY_KEY:-your-encrpytion-primary-key}
       - ENCRYPTION_DETERMINISTIC_KEY=${ENCRYPTION_DETERMINISTIC_KEY:-your-encrpytion-deterministic-key}
       - ENCRYPTION_KEY_DERIVATION_SALT=${ENCRYPTION_KEY_DERIVATION_SALT:-your-encrpytion-derivation-salt}
-      - LAGO_USE_AWS_S3=${LAGO_USE_AWS_S3}
-      - LAGO_AWS_S3_ACCESS_KEY_ID=${LAGO_AWS_S3_ACCESS_KEY_ID}
-      - LAGO_AWS_S3_SECRET_ACCESS_KEY=${LAGO_AWS_S3_SECRET_ACCESS_KEY}
-      - LAGO_AWS_S3_REGION=${LAGO_AWS_S3_REGION}
-      - LAGO_AWS_S3_BUCKET=${LAGO_AWS_S3_BUCKET}
+      - LAGO_USE_AWS_S3=${LAGO_USE_AWS_S3:-false}
+      - LAGO_AWS_S3_ACCESS_KEY_ID=${LAGO_AWS_S3_ACCESS_KEY_ID:-}
+      - LAGO_AWS_S3_SECRET_ACCESS_KEY=${LAGO_AWS_S3_SECRET_ACCESS_KEY:-}
+      - LAGO_AWS_S3_REGION=${LAGO_AWS_S3_REGION:-}
+      - LAGO_AWS_S3_BUCKET=${LAGO_AWS_S3_BUCKET:-}
       - LAGO_PDF_URL=${LAGO_PDF_URL:-http://pdf:3000}
       - LAGO_REDIS_CACHE_URL=redis://redis:6379
-      - SEGMENT_WRITE_KEY=${SEGMENT_WRITE_KEY}
-      - LAGO_DISABLE_SEGMENT=${LAGO_DISABLE_SEGMENT}
```

Comment:
Why removing this one?

Reply:
it's not used on the other `api` based service, is it needed?

Reply:
It's used here: https://github.com/getlago/lago-api/blob/main/config/initializers/analytics_ruby.rb#L1